lib/generators/unread/migration/templates/migration.rb

```diff
@@ -1,16 +1,16 @@
 class UnreadMigration < Unread::MIGRATION_BASE_CLASS
   def self.up
-    create_table ReadMark, force: true, options: create_options do |t|
+    create_table :read_marks, force: true, options: create_options do |t|
```

Comment:
I am unfamiliar with the `create_table ClassName` syntax and couldn't find good docs on it. However the specs failed and did not properly add the `read_at` column to the table until I changed this.

Maybe you know more about why you chose this syntax?